core/src/test/java/org/jdbi/v3/core/TestHandle.java

```diff
@@ -204,15 +204,12 @@ public void testUseHandleUsesSameHandle() {
     @Test
     public void testAllNestedOpsSameHandle() {
         Jdbi jdbi = h2Extension.getJdbi();
-        jdbi.useHandle(handle1 ->
-                jdbi.useTransaction(handle2 ->
-                        jdbi.withHandle(handle3 ->
-                                jdbi.inTransaction(handle4 -> {
-                                assertThat(handle1).isSameAs(handle2);
-                                assertThat(handle1).isSameAs(handle3);
-                                assertThat(handle1).isSameAs(handle4);
-                                return null;
-                            }))));
+        jdbi.useHandle(handle1 -> jdbi.useTransaction(handle2 -> jdbi.withHandle(handle3 -> jdbi.inTransaction(handle4 -> {
```

Comment:
I consider the multi-line variant much more readable than the one line variant.

Reply:
I agree with you. The issue with the original code was indentation width of 8 (should be 4).